Source/UserManagement/Read/GreetingGenerators/GreetingHistoryEventProcessor.cs

```diff
@@ -11,17 +11,18 @@ public GreetingHistoryEventProcessor(IGreetingHistories greetingHistories)
         {
             _greetingHistories = greetingHistories;
         }
-
-        public void Process(PhoneNumberAddedToDataCollector @event)
+        
+        //TODO: QUESTION: Shouldn't this listen to MessageGenerated-event?
```

Comment:
I'm not sure I follow what's happening here.

If we want to generate a message when a phone number is added and then track the success of the message (know if it arrived), then I think the "MessageGenerated" event should contain more and should be the one that is tracked.